### PR TITLE
Improve logger a bit

### PR DESCRIPTION
**Pull request by Simon-Laux**
- omit stacktrace for debug and info
- move core event logs to their own log channel (makes event output smaller and less annoying)
- change logile extention to .log.tsv
- add colors, time elapsed and generally rework main process logging a bit:

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -26,9 +28,28 @@ function log (channel, level, ...args) {
...
+    if (isMainProcess) {
+      const begining = `${Math.round((Date.now() - startTime) / 100) / 10}s ${[
+        '[D]',
+        colors.blue('[i]'),
+        colors.yellow('[w]'),
+        colors.red('[E]'),
+        colors.red('[C]')
+      ][level]}${colors.grey(channel)}:`
+      if (!stacktrace) {
+        console.log(begining, ...args)
```
Comment (Jikstra) [inline src/logger.js]: `/* ignore-console-log */` is missing

File: src/logger.js [hunk 2]
```diff
@@ -26,9 +28,28 @@ function log (channel, level, ...args) {
...
+      } else {
+        console.log(begining, ...args, colors.red(stacktrace))
```
Comment (Jikstra) [inline src/logger.js]: same

**Comment by Simon-Laux** [comment]
> Very helpful. You said that core events are now in their own channel, how can i still see them?

With channel I mean channel of the logger. At the moment there is no filtering going on at all so all channels are displayed. We might develop a log reader that can filter the channels of the logfiles to make reading them easier.
At the moment the change is:
before:
```
main/deltachat DEBUG [] Core Event DC_EVENT_INFO  IMAP-IDLE SPAWNING
```
after:
```
core/event DEBUG [] DC_EVENT_INFO  IMAP-IDLE SPAWNING
```
after all changes:
```
3.2s [D]core/event: DC_EVENT_INFO  IMAP-IDLE SPAWNING
```

**Pull request merged**